tests/test_create.py

```diff
@@ -22,3 +25,20 @@ def test_create_environment_from_lockfile_pixi_metadata(tmp_path: Path) -> None:
     assert env_record_path.is_file()
     data = json.loads(env_record_path.read_bytes())
     assert data["license"] == "ONLY_IN_LOCKFILE"
+
+
+@pytest.mark.skipif(
+    condition=sys.platform.startswith("win"),
```

Comment:
we already import a bunch from conda, might as well just reuse `conda.common.compat.on_win`